firebase/firebase.function.ts

```diff
@@ -173,6 +175,44 @@ export const createGameDoc = async(game : GameInfo): Promise<string> => {
 
 }
 
+
+async function deleteCollection(db, collectionPath, batchSize) {
```

Comment:
Yeah I definitely messed up there. I added this hella long time ago. Its a cloud funciton I think, provided to us by the firebase firestore. If I were to use this, I should definitely be doing this inside NodeJS or some server side environment? (If my research is correct at least xd)

Reply:
The purpose of this delete by the way, is to ensure that when the user leaves the game, their files get deleted from the gamedoc as well. I believe that somewhere along the way the user's files include collections, so that needs to be deleted as well.

Id love to know if there's any better way to deal with users leaving the game. 

Also the users can't leave the game just yet I hadn't even added the button that would route them back xd. But yeah it'll happen.